h/collections.h

```diff
@@ -259,9 +259,52 @@ ZOWE_PRAGMA_PACK_RESET
 #ifndef QueueAmode64
 #define QueueAmode64
 #endif
+
+/**
+ * @brief Make a lock-free queue.
+ * @param flags Unused at the moment.
+ * @return Pointer to the queue.
+ */
 Queue *makeQueue(int flags) QueueAmode64;
+
+/**
+ * @brief Destroy a lock-free queue.
+ * @param q Queue to be destroyed.
+ */
 void destroyQueue(Queue *q) QueueAmode64;
+
+#ifdef __ZOWE_OS_ZOS
+
+/**
+ * Enqueue a queue element.
+ * @param q Queue to which the new element will be enqueued.
+ * @param newElement Queue element to be enqueued. WARINIG: it must reside on
+ * a double word boundary, otherwise a specification exception (S0C6 ABEND)
```

Comment:
I'm not sure if double word alignment is required (ABEND0C6) or recommended for performance reasons. Maybe Greg can comment?